Makefile

```diff
@@ -104,6 +107,17 @@ server-static: build-web
 		upx --best --lzma $(BIN_OUTPUT_PATH)/viam-server;\
 	fi
 
+ndk-root:
+	echo $(NDK_ROOT)
+
+bin/viam-server-android-$(SERVER_RELEASE):
```

Comment:
So while normally using "real" targets is a good idea in Makefiles, it doesn't play nice with go, generally speaking. If someone runs `make bin/viam-server-android-local` once, then changes some code, and runs it again, the second time it'll do nothing, because the target will already exist. At the same time, depending on every go file in the entire repo isn't practical. So all our build targets are typically .PHONY ones (even if we don't actually properly spec them with the .PHONY label everywhere yet.) 

So renaming this just `server-android` (to match `server-static`) would probably be best.

Reply:
Also, for naming, I think it'd be good to include the cpu architecture as well, as I suspect we may end up having to do some 32-bit builds at some point. Ex: `viam-server-$(BUILD_CHANNEL)-android-aarch64` (assuming `uname -m` returns "aarch64" on android. Whatever that returns.)

Reply:
added aarch64 in e6615931